examples/connectivity/plot_canica_resting_state.py

```diff
@@ -46,11 +47,16 @@
 components_img.to_filename('canica_resting_state.nii.gz')
 
 ### Visualize the results #####################################################
-# Show some interesting components
+
 import matplotlib.pyplot as plt
+from nilearn.plotting import plot_prob_atlas
 from nilearn.plotting import plot_stat_map
 from nilearn.image import iter_img
 
+# Plot all ICA components
+plot_prob_atlas(components_img, title='All ICA components')
+
+# Show some interesting components
```

Comment:
Actually, we are showing all the components here. So I think that the comment needs to be changed.